src/codegen/templates/_grpc_interpreter.py.mako

```diff
@@ -0,0 +1,58 @@
+<%
+    from codegen.utilities.interpreter_helpers import get_interpreter_functions, get_interpreter_parameter_signature, get_output_parameter_names, get_reponse_parameters
+    from codegen.utilities.function_helpers import order_function_parameters_by_optional
+    from codegen.utilities.text_wrappers import wrap, docstring_wrap
+    from codegen.utilities.helpers import snake_to_pascal
+    functions = get_interpreter_functions(data)
```

Comment:
You can't instantiate `GrpcStubInterpreter` because it doesn't override all of the methods in the abstract base class:

```
Traceback (most recent call last):
  File "D:\dev\nidaqmx-python\examples\ai_voltage_sw_timed.py", line 16, in <module>
    interpreter = nidaqmx._grpc_interpreter.GrpcStubInterpreter(GrpcOptions(grpc_channel))
TypeError: Can't instantiate abstract class GrpcStubInterpreter with abstract methods cfg_anlg_multi_edge_ref_trig, cfg_anlg_multi_edge_start_trig, cfg_input_buffer, cfg_output_buffer, cfg_time_start_trig, create_watchdog_timer_task, delete_saved_global_chan, delete_saved_scale, delete_saved_task, device_supports_cal, get_ai_chan_cal_cal_date, get_ai_chan_cal_exp_date, get_buffer_attribute_uint32, get_cal_info_attribute_bool, get_cal_info_attribute_double, get_cal_info_attribute_string, get_cal_info_attribute_uint32, get_chan_attribute_bool, get_chan_attribute_double, get_chan_attribute_double_array, get_chan_attribute_int32, get_chan_attribute_string, get_chan_attribute_uint32, get_device_attribute_bool, get_device_attribute_double, get_device_attribute_double_array, get_device_attribute_int32, get_device_attribute_int32_array, get_device_attribute_string, get_device_attribute_uint32, get_device_attribute_uint32_array, get_error_string, get_exported_signal_attribute_bool, get_exported_signal_attribute_double, get_exported_signal_attribute_int32, get_exported_signal_attribute_string, get_exported_signal_attribute_uint32, get_nth_task_channel, get_nth_task_device, get_nth_task_read_channel, get_persisted_chan_attribute_bool, get_persisted_chan_attribute_string, get_persisted_scale_attribute_bool, get_persisted_scale_attribute_string, get_persisted_task_attribute_bool, get_persisted_task_attribute_string, get_physical_chan_attribute_bool, get_physical_chan_attribute_bytes, get_physical_chan_attribute_double, get_physical_chan_attribute_double_array, get_physical_chan_attribute_int32, get_physical_chan_attribute_int32_array, get_physical_chan_attribute_string, get_physical_chan_attribute_uint32, get_physical_chan_attribute_uint32_array, get_read_attribute_bool, get_read_attribute_double, get_read_attribute_int32, get_read_attribute_string, get_read_attribute_uint32, get_read_attribute_uint64, get_real_time_attribute_bool, get_real_time_attribute_int32, get_real_time_attribute_uint32, get_scale_attribute_double, get_scale_attribute_double_array, get_scale_attribute_int32, get_scale_attribute_string, get_self_cal_last_date_and_time, get_system_info_attribute_string, get_system_info_attribute_uint32, get_task_attribute_bool, get_task_attribute_string, get_task_attribute_uint32, get_timing_attribute_bool, get_timing_attribute_double, get_timing_attribute_ex_bool, get_timing_attribute_ex_double, get_timing_attribute_ex_int32, get_timing_attribute_ex_string, get_timing_attribute_ex_uint32, get_timing_attribute_ex_uint64, get_timing_attribute_int32, get_timing_attribute_string, get_timing_attribute_uint32, get_timing_attribute_uint64, get_trig_attribute_bool, get_trig_attribute_double, get_trig_attribute_double_array, get_trig_attribute_int32, get_trig_attribute_int32_array, get_trig_attribute_string, get_trig_attribute_uint32, get_watchdog_attribute_bool, get_watchdog_attribute_double, get_watchdog_attribute_int32, get_watchdog_attribute_string, get_write_attribute_bool, get_write_attribute_double, get_write_attribute_int32, get_write_attribute_string, get_write_attribute_uint32, get_write_attribute_uint64, read_power_binary_i16, read_power_f64, read_raw, register_done_event, register_every_n_samples_event, register_signal_event, reset_buffer_attribute, reset_chan_attribute, reset_exported_signal_attribute, reset_read_attribute, reset_real_time_attribute, reset_timing_attribute, reset_timing_attribute_ex, reset_trig_attribute, reset_watchdog_attribute, reset_write_attribute, save_global_chan, save_scale, save_task, self_cal, set_ai_chan_cal_cal_date, set_ai_chan_cal_exp_date, set_buffer_attribute_uint32, set_cal_info_attribute_bool, set_cal_info_attribute_double, set_cal_info_attribute_string, set_cal_info_attribute_uint32, set_chan_attribute_bool, set_chan_attribute_double, set_chan_attribute_double_array, set_chan_attribute_int32, set_chan_attribute_string, set_chan_attribute_uint32, set_exported_signal_attribute_bool, set_exported_signal_attribute_double, set_exported_signal_attribute_int32, set_exported_signal_attribute_string, set_exported_signal_attribute_uint32, set_read_attribute_bool, set_read_attribute_double, set_read_attribute_int32, set_read_attribute_string, set_read_attribute_uint32, set_read_attribute_uint64, set_real_time_attribute_bool, set_real_time_attribute_int32, set_real_time_attribute_uint32, set_scale_attribute_double, set_scale_attribute_double_array, set_scale_attribute_int32, set_scale_attribute_string, set_timing_attribute_bool, set_timing_attribute_double, set_timing_attribute_ex_bool, set_timing_attribute_ex_double, set_timing_attribute_ex_int32, set_timing_attribute_ex_string, set_timing_attribute_ex_uint32, set_timing_attribute_ex_uint64, set_timing_attribute_int32, set_timing_attribute_string, set_timing_attribute_uint32, set_timing_attribute_uint64, set_trig_attribute_bool, set_trig_attribute_double, set_trig_attribute_double_array, set_trig_attribute_int32, set_trig_attribute_int32_array, set_trig_attribute_string, set_trig_attribute_uint32, set_watchdog_attribute_bool, set_watchdog_attribute_double, set_watchdog_attribute_int32, set_watchdog_attribute_string, set_write_attribute_bool, set_write_attribute_double, set_write_attribute_int32, set_write_attribute_string, set_write_attribute_uint32, set_write_attribute_uint64, wait_for_next_sample_clock, wait_for_valid_timestamp, write_raw
```

This is a temporary problem. You can work around it by specifying `is_base_interpreter=True`. This will generate some broken methods, but part of the class will work.

Reply:
For now I have generated the additional methods as per the suggestion, but this will be removed in a separate PR where we will update the `base_interpreter.py` with only the required functions.